src/extensions/AdditiveColormap3DExtensions/additive-colormap.glsl

```diff
@@ -0,0 +1,49 @@
+#pragma glslify: jet = require("glsl-colormap/jet")
```

Comment:
These are all copied from the 2D colormap as well, right? I can think about it a bit more, but it's making me think this is best suited for a build-step.

Reply:
Yeah.